extensions-twitter/scribe-emitter/src/main/java/org/apache/druid/emitter/scribe/ScribeAdminLogEntry.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.emitter.scribe;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.druid.java.util.emitter.core.Event;
+import org.apache.druid.java.util.emitter.service.ServiceMetricEvent;
+import org.apache.thrift.TBase;
+
+public class ScribeAdminLogEntry
+{
+  private static final String AUDIT_KEY = "key";
+  private static final String AUDIT_TYPE = "type";
+  private static final String AUTHOR = "author";
+  private static final String COMMENT = "comment";
+  private static final String IP = "remote_address";
+  private static final String CREATED_DATE = "created_date";
+  private static final String PAYLOAD = "payload";
+
+  private String audit_key;
+  private String audit_type;
+  private String author;
+  private String comment;
+  private String remote_address;
+  private String created_date;
+  private String payload;
+
+  private String role;
+  private String druid_version;
+  private String environment;
+  private String dataCenter;
+  private String cluster_name;
+
+  public ScribeAdminLogEntry(Event event, ScribeEmitterConfig config, ObjectMapper jsonMapper) throws JsonProcessingException
+  {
+    ServiceMetricEvent adminLogEvent = (ServiceMetricEvent) event;
+
+    this.audit_key = (String) adminLogEvent.getUserDims().getOrDefault(AUDIT_KEY, null);
```

Comment:
do we need this cast: (String) ?

Reply:
I think we need this case as `getUserDims()` returns a `Map<String, Object>`

Reply:
get it